src/server/context.ts

```diff
@@ -55,9 +57,22 @@ export class ContextManager implements IContext {
     };
     this.models = {
       tag: new TagModel(this),
+      pocketSave: new PocketSaveModel(this),
     };
   }
-  models: { tag: TagModel };
+  models: {
+    tag: TagModel;
+    pocketSave: PocketSaveModel;
+  };
+
+  withDbClientOverride(dbClient: Knex): ContextManager {
```

Comment:
with our new change on choosing dbClient at ContextManager based on request, we don't need this function right?

Reply:
@sri-kirsh this is mess i picked up from your changes I believe. I'll repair it shortly.